html/about.html

```diff
@@ -0,0 +1,144 @@
+            So if you're active and engaged in the worlds of open ocntent and knowledge - free 
+            software advocates, Wikipedians, Open Knowledge, galleries, libraries, museums, 
+            archives, governments and foundations, lawyers and activits - we hope you consider 
+            joining us this year to build a stronger, more vibrant commons together.
+          </span>
+        </p>
+        <article id="contact">
+          <span>
+            Please contact us via email for any further questions
+            <br>
+            about CC Global Summit 2015!
+          </span>
+          <br>
+          <a href="#">ccsummit2015@cckorea.org</a>
+        </article>
+      </section>
+    </main>
+    <section id="about-logo" class="section">
+      <div class="section-context">
+        <h2>
+          Creative Commons 
+          <br>
+          Global Summit 2015 Logo
+        </h2>
+        <hr class="hr">
+        <span>
+          The logo of CC Global Summit 2015 was decided through the logo 
+          competition from June 8th to July 7th.
+        </span>
+        <span class="desktop-visible">
+          After online public voting and Selection Comitee's judging process, 
+          the logo from Naresh Agrawal won the prize.
+          <br>
+          After consultation with Mr. Agrawal we optimized the prize-winning-
+          logo for this year's summit as following:
+        </span>
+      </div>
+      <img src="../assets/images/about_logo.png" alt="CC Global Summit 2015 Logo">
+    </section>
+    <hr class="sections-hr">
+    <section class="section" id="about-summits">
+      <div class="section-context" id="summit-top">
+        <h2>
+          See the past CC Global Summits
+        </h2>
+        <hr class="hr">
+        <span>
+          Take a loot at the last two CC Global Summits which took place 
+          in Buenos Aires and in Warsaw.
+        </span>
+      </div>
+      <img id="summit13" class="summit-img" src="../assets/images/summit2013.png" alt="2013 Global Summit Buenos Aires" tabindex="0">
+      <img id="summit11" class="summit-img" src="../assets/images/summit2011.png" alt="2011 Global Summit Warsaw" tabindex="0">
+    </section>
+    <section id="about-partners" class="section dark">
+      <div class="section-context">
+        <h2 id="partners-h2">
+          Partners
+        </h2>
+        <hr class="hr">
+        <ul id="sponsors">
+          <li><img class="sponsor-img" src="../assets/images/mozilla.png" alt=""></li>
+          <li><img class="sponsor-img" src="../assets/images/google.png" alt=""></li>
+          <li><img class="sponsor-img" src="../assets/images/naver.png" alt=""></li>
+          <li><img class="sponsor-img" src="../assets/images/kakao.png" alt=""></li>
+          <li><img class="sponsor-img" src="../assets/images/airbnb.png" alt=""></li>
+        </ul>
+      </div>
+    </section>
+    <footer id="about-license" class="container-fullwidth rights">

```

Comment:
- [ ] Your footer is almost ready, please align the elements and add padding and margins following the template:

Template:
![Captura de pantalla 2022-04-14 a la(s) 8 18 12 p  m](https://user-images.githubusercontent.com/83717285/163502726-88aa7c83-a7df-45c5-ae82-96b6b4f5e3fe.png)

Your design:
![Captura de pantalla 2022-04-14 a la(s) 8 18 24 p  m](https://user-images.githubusercontent.com/83717285/163502752-d3aa6c6d-9d7b-48b6-a1ba-79826361098c.png)